### PR TITLE
add timespan parsing

### LINKED ISSUE #37: error parsing these files
sorry i keep giving you stuff to fix :|
[File 1](https://github.com/localcc/gvas/files/12445849/File.1.zip) fails to parse with `Invalid string size -1171893120 at position 2071`

### PR DESCRIPTION
**Pull request by bananaturtlesandwich**
fixes #37

### CODE REVIEW

File: src/properties/struct_property.rs [hunk 1]
```diff
@@ -176,6 +178,9 @@ impl StructProperty {
             "DateTime" => StructPropertyValue::DateTime(DateTime {
                 ticks: UInt64Property::read(cursor, false)?.value,
             }),
+            "Timespan" => StructPropertyValue::Timespan(DateTime {
```
Comment (scottanderson) [inline src/properties/struct_property.rs:181]: I would suggest making a Timestamp struct

Comment (localcc) [reply src/properties/struct_property.rs:181]: shoudn't this follow the same rules as DateTime?

Comment (bananaturtlesandwich) [reply src/properties/struct_property.rs:181]: the representation is the exact same - internally they're both referred to as ticks so I guess I could maybe rename the struct to Time or smthn

Comment (bananaturtlesandwich) [reply src/properties/struct_property.rs:181]: > shouldn't this follow the same rules as DateTime?

it needs a different property name which is why I didn't just | "Timespan"

Comment (localcc) [reply src/properties/struct_property.rs:181]: yeah this was a question to Scott

**Pull request merged**